src/AstroPanda.Blazor.Toolkit/Services/ContextMenu/ContextMenuReference.cs

```diff
@@ -0,0 +1,33 @@
+﻿using AstroPanda.Blazor.Toolkit.Components;
+using Microsoft.AspNetCore.Components;
+
+namespace AstroPanda.Blazor.Toolkit.Services;
```

Comment:
```suggestion
namespace AstroPanda.Blazor.Toolkit;
```
Let's simplify the namespace for those items that are publicly available to consumers